src/main/java/pl/wedding/management/service/weddinghall/model/AddressHall.java

```diff
@@ -0,0 +1,17 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
```

Comment:
NoArgsConstructor has no sense. You receive empty object, with null fields, and won't be able to set any (No setter).
You can use @Value designed for immutability, and please add @Builder. It simplifies creating objects.